api/openapi/authentication.md

```diff
@@ -20,6 +20,8 @@ curl --header "Authorization: Bearer 1a6a9936ad150a2ee345c65331da7a3ccc2de" http
 
 By default, API keys are only generated for admins, but you can easily customize Solidus to generate them for all users, which is useful for instance if you want users to be able to sign in and manage their profile via the API.
 
+The `API key` is mandatory for each endpoint by default. You can change this configuration [here](https://github.com/solidusio/solidus/blob/master/api/lib/spree/api_configuration.rb#L5) if you want to avoid the default behaviour exposing some endpoints without an API key. An example is the [GET product list](https://solidus.stoplight.io/docs/solidus/08307f3d809e7-list-products).
```

Comment:
Please use the [GitHub permanent link](https://docs.github.com/en/repositories/working-with-files/using-files/getting-permanent-links-to-files) to this line. If it changes, the documentation will point to another thing otherwise.